core/pen/include/pen.h

```diff
@@ -46,8 +46,8 @@ namespace pen
 
     struct pen_creation_params
     {
-        volatile u32     window_width = 1280;
-        volatile u32     window_height = 720;
+        volatile f32     window_width = 1280.0f;
```

Comment:
how come this needed to change from `u32` to `f32`? Just wondering now about type conversion warnings that may occur and all of the samples initialising this struct needing to change.

Would prefer this not to change in this PR if possible. If there is a reason we can do it in a separate pass.

Reply:
Ahh I thought I had reverted that. It was an update on the ImGui I was confused on will change it back.

Reply:
reverted